packages/python/beagle/beagle/plan/yarn.py

```diff
@@ -37,6 +38,7 @@ def __init__(
         )
 
         self.source_path = source_path
+        self.yarn_workspace = yarn_workspace,
```

Comment:
Will this have upstream effects? If so we should have an accompanying MR on the BHE side.

Reply:
The BHE side is in progress